distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+[ 111] [Active     ] [            ] [       ] [   80] Commons IO (2.1.0)
+[ 113] [Active     ] [Created     ] [       ] [   80] ${branding} :: Distribution :: Console Branding Plugin (${project.version})
+----
+
+.${branding} Application Installation Dependencies
+
+[WARNING]
+====
+If completing a non-standard installation, be aware that some applications depend on other ${branding} applications being installed.
+
+This hierarchy can be shown using the `app:tree` command
+
+[source]
+----
+${ddf-branding-lowercase}${at-symbol}local>app:tree
++- opendj-embedded
++- platform-app
+|   +- catalog-app
+|   |   +- search-ui-app
+|   |   |   +- spatial-app
+|   |   +- solr-app
+|   +- security-services-app
+|   |   +- admin-app
+----
+
+====
+
+==== Verifying Installation
+
+At this point, ${branding} should be configured and running with a Solr Catalog Provider.
+New features (endpoints, services, and sites) can be added as needed.
+
+Verification is achieved by checking that all of the ${branding} bundles are in an Active state (excluding fragment bundles which remain in a Resolved state).
+
+Execute the following command to display the status of all the ${branding} bundles:
+
+.View Status
+----
+${branding-lowercase}${at-symbol}local>list  grep -i ${branding-lowercase}
+
+----
+
+[WARNING]
+====
+Entries in the *Resolved* state are expected, they are OSGi bundle fragments.
+Bundle fragments are distinguished from other bundles in the command line console list by a field named `Hosts`, followed by a bundle number.
+Bundle fragments remain in the *Resolved* state and can never move to the *Active* state.
+====
+
+.Example: Bundle Fragment in the Command Line Console
+----
+215 | Resolved |  80 | ${project.version}     | DDF :: Platform :: Security :: Session, Hosts: 90
+----
+
+After successfully completing these steps, the ${branding} is ready to be configured.
+
+==== ${branding} Directory Contents after Installation and Initial Startup
+
+During ${branding} installation, the major directories and files shown in the table below are created, modified, or replaced in the destination directory.
+
+.${branding} Directory Contents
+[cols="1,4" options="header"]
+|===
+
+|Directory Name
+|Description
+
+|`bin`
+|Scripts to start and stop ${branding}
+
+|`data`
+|The working directory of the system – installed bundles and their data
+
+|`data/log/${branding-lowercase}.log`
+|Log file for ${branding}, logging all errors, warnings, and (optionally) debug statements. This log rolls up to 10 times, frequency based on a configurable setting (default=1 MB)
+
+|`data/log/ingest.log`
+|Log file for any ingest errors that occur within ${branding}.
+
+|`data/log/security.log`
+|Log file that records user interactions with the system for auditing purposes.
+
+|`deploy`
+|Hot-deploy directory – KARs and bundles added to this directory will be hot-deployed (Empty upon ${branding} installation)
+
+|`documentation`
```

Comment:
Not sure what happened. Must have been a bad build on my end cause it was ok last time I built. Ignore for now.